typing-element.js

```diff
@@ -41,6 +88,13 @@ class TypingElement extends AnimationElement{
         this.#span.innerHTML = typedText
         this.enableAnimation()
     }
+
+    /** 
+     * Resets TypingElement object
+     * @property {function} reset
+     * @member {Object}
+     * @returns {number} Amount of Typed Letters in object
```

Comment:
This one sets typed letters to 0, but does not return anything. We could also omit that one